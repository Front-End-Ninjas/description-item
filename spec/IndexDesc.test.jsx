import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../react-client/src/IndexDesc';

describe('Index Component', () => {
  it('should be defined', () => {
    expect(Index).toBeDefined();
  });

  it('should renders correctly', () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
