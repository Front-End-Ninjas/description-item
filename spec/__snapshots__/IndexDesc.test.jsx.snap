// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Component should renders correctly 1`] = `
<div
  id="main_app"
>
  <div
    id="main_desc"
  >
    <div>
      <p
        id="brand"
      >
        Amazon
      </p>
    </div>
    <div>
      <h1
        id="model"
      >
        Apple TV - 32GB (4th Generation)
      </h1>
    </div>
    <div>
      <p
        id="price_shipping"
      >
        Price: 
        <span
          id="price"
        >
           $
          1359
        </span>
        <span
          id="shipping"
        >
           
          <img
            alt="Amazon Prime"
            src="../prime_logo.png"
          />
        </span>
      </p>
    </div>
    <div>
      <h3
        id="stock"
      >
        In Stock.
      </h3>
    </div>
    <div
      id="list_desc"
    >
      <div>
        <ul>
          <li
            id="list-item"
          >
            Maecenas congue leo at tempor lacinia.
          </li>
        </ul>
      </div>
      <div>
        <ul>
          <li
            id="list-item"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </li>
        </ul>
      </div>
      <div>
        <ul>
          <li
            id="list-item"
          >
            Cras ut sem vel lectus rhoncus imperdiet a at leo.
          </li>
        </ul>
      </div>
      <div>
        <ul>
          <li
            id="list-item"
          >
            Etiam a risus luctus, lacinia dui in, varius lorem.
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
